sopel/modules/unicode_info.py

```diff
@@ -10,11 +10,14 @@
 
 from sopel import plugin
 
-# unicodedata2 can provide a more-modern UCD than the one that comes with Python, use it if present
+# Python built-in unicodedata uses UCD version 13 (as of Python 3.10)
+# unicodedata2 can provide a more recent version, so we use that if present
+# See also: https://docs.python.org/fr/3/library/unicodedata.html
```

Comment:
Did you mean to link the French documentation set here?

```suggestion
# See also: https://docs.python.org/3/library/unicodedata.html
```

Reply:
Yes?

(absolutely not, oops)